tests/gtest/avif_fuzztest_helpers.h

```diff
@@ -92,20 +108,67 @@ inline auto ArbitraryAvifImage16b() {
             fuzztest::Just(has_alpha),
             fuzztest::ContainerOf<std::vector<uint16_t>>(
                 fuzztest::InRange<uint16_t>(0, (1 << depth) - 1))
-                .WithSize(
-                    GetNumSamples(width, height, pixel_format, has_alpha)));
+                .WithSize(GetNumSamples(/*num_frames=*/1, width, height,
+                                        pixel_format, has_alpha)));
       },
       fuzztest::InRange<uint16_t>(1, kMaxDimension),
       fuzztest::InRange<uint16_t>(1, kMaxDimension),
       fuzztest::ElementOf({10, 12}), ArbitraryPixelFormat(),
       fuzztest::Arbitrary<bool>());
 }
 
+// avifImage generator type: Number of frames, width, height, pixel format and
+// 8-bit samples.
+inline auto ArbitraryAvifAnim8b() {
+  return fuzztest::FlatMap(
+      [](size_t num_frames, size_t width, size_t height,
+         avifPixelFormat pixel_format, bool has_alpha) {
+        return fuzztest::Map(
+            CreateAvifAnim8b, fuzztest::Just(num_frames), fuzztest::Just(width),
+            fuzztest::Just(height), fuzztest::Just(pixel_format),
+            fuzztest::Just(has_alpha),
+            fuzztest::Arbitrary<std::vector<uint8_t>>().WithSize(GetNumSamples(
+                num_frames, width, height, pixel_format, has_alpha)));
```

Comment:
That will force all frames to have alpha or none to have alpha. Maybe we should have a vector of bools?
Same for width and height but I actually do not know whether AVIF supports animations with frames of different size: maybe you can run this test locally to see if it passes with frames of different size?